foundations/javascript_basics/revisiting_rock_paper_scissors.md

```diff
@@ -1,17 +1,50 @@
 ### Introduction
 
-Now that we can manipulate the DOM, it's time to revisit [Rock Paper Scissors](https://www.theodinproject.com/paths/foundations/courses/foundations/lessons/rock-paper-scissors) and add a simple UI to it. Before we proceed, we need to get familiar with a very popular Git workflow. 
+Now that we can manipulate the DOM, it's time to revisit [Rock Paper Scissors](https://www.theodinproject.com/paths/foundations/courses/foundations/lessons/rock-paper-scissors) and add a simple UI to it. Before we proceed, we need to get familiar with a very popular Git workflow.
+
+Before you start making changes to your Rock Paper Scissors project, you need to learn about a concept in Git called **branching** so that you can make your changes without having to worry about breaking what you have now.
+
+Branches in Git allow your repository to hold multiple *alternate reality* versions of your files at the same time. You’ve actually (sort of) been using branches since you made your first commit, you just might not have known it! Back in [the setting up Git lesson](https://www.theodinproject.com/paths/foundations/courses/foundations/lessons/setting-up-git) when you ran `git config --global init.defaultBranch main` you were setting the name of what’s called the *default* branch for your repos. The default branch is just what we call the branch that is created when you make your first commit on a project, and in that command we set the name to be `main` as is the current standard. 
```

Comment:
I went with alternate reality because I thought it would make the content a bit less dry, but if you think this is a major clarity issue I could be persuaded away from that decision @mgrigoriev8109

Reply:
It's not major by any means, reading through it a second time I think people shouldn't have any trouble understanding it.